mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/ThreadMirror.cs

```diff
@@ -30,11 +30,20 @@ public class ThreadMirror : ObjectMirror
 			return frames;
 		}
 
+		public long ElapsedTime () {
+			long elapsedTime = GetElapsedTime ();
```

Comment:
How come this is split into also an internal method (ie. why not `return vm.conn.Thread_GetElapsedTime (id)`)?

Reply:
I don't have a good reason I was following the other implementations, but I really think that this is an unnecessary overhead. Should I change this and do a new PR?